Content.Client/Storage/Systems/EntityStorageSystem.cs

```diff
@@ -39,4 +42,9 @@ public override bool ResolveStorage(EntityUid uid, [NotNullWhen(true)] ref Share
         component = storage;
         return component != null;
     }
+
+    private void OnStatusIconVisible(EntityUid uid, InsideEntityStorageComponent comp, ref StatusIconVisibleEvent args)
+    {
+        args.Visible = false;
+    }
 }
```

Comment:
This event isn't recursive so won't this break?

Reply:
in what case? invisible person inside a bag in a locker????

Reply:
yes

Reply:
what component would be ideal for doing the recursion

EntityStorage? Storage? both?

Reply:
I don't think either mark InsideEntityStorage recursively, you would need to do it on the storage itself probably.